xbmc/utils/log.cpp

```diff
@@ -272,6 +272,21 @@ spdlog::level::level_enum CLog::MapLogLevel(int level)
   return spdlog::level::info;
 }
 
+void CLog::FormatAndLogInternal(spdlog::level::level_enum level,
+                                fmt::string_view format,
+                                fmt::format_args args)
+{
+  if (level < m_defaultLogger->level())
+    return;
+
+  auto message = fmt::vformat(format, args);
+
+  // fixup newline alignment, number of spaces should equal prefix length
+  FormatLineBreaks(message);
```

Comment:
`FormatLineBreaks()` only is used here, then maybe remove this method and call directly:

`StringUtils::Replace(message, "\n", "\n                                                   ");`